src/components/Restoration/YourCompanySummary.vue

```diff
@@ -41,12 +41,18 @@
 </template>
 
 <script lang="ts">
+import { ApprovalTypes } from '@bcrs-shared-components/enums'
 import { mapGetters } from 'vuex'
 
 export default {
+  data () {
+    return {
+      ApprovalTypes: ApprovalTypes
```

Comment:
Nice. I bet you had to search how to do this 👍

Reply:
Indeed I had to Sev! 😸

Reply:
By the way, when you see `x: x` in an object, you can simplify it to just `x`.

Reply:
Ohh, Cool! Didn't know that. Thanks Sev!